source/features/clean-conversation-sidebar.tsx

```diff
@@ -46,7 +45,10 @@ function cleanSection(selector: string): boolean {
 		'[aria-label="Select projects"] .Link--primary',
 	];
 
-	const heading = select('.discussion-sidebar-heading', container)!;
+	const heading = select([
+		'details:has(> .discussion-sidebar-heading)', // Can edit sidebar, has a dropdown
+		'.discussion-sidebar-heading', // Cannot editor sidebar, has a plain heading
+	], container)!;
```

Comment:
I couldn't use `selectHas` because it doesn't support selector lists and it's not worth working on that anymore. Firefox support will follow eventually.